packages/optimistic-oracle/src/proposer.js

```diff
@@ -118,6 +121,7 @@ class OptimisticOracleProposer {
   async settleRequests() {
     this.logger.debug({
       at: "OptimisticOracleProposer#settleRequests",
+      oracleType: OptimisticOracleType[this.optimisticOracleClient.oracleType],
```

Comment:
Could we swap all these for just `oracleType: this.optimisticOracleClient.oracleType` here and in the rest of the logs (looks like there are about 9 spots)?

Reply:
Oh shoot thought I did this, will fix